### DISCUSSION TITLE
萨卡兹肉鸽一进二层就卡住

### DISCUSSION THREAD

**Discussion by Morrri725** [MaaAssistantArknights/MaaAssistantArknights#11040] [General]
![萨卡兹](https://github.com/user-attachments/assets/16fbdba5-132d-44e2-9bdb-1f125b6f3407)
每次从一层末尾的商店出来，进到第二层，到这个界面的时候就不动了
<img width="904" alt="image" src="https://github.com/user-attachments/assets/2f88c96c-c142-45d7-8bbb-889fc3bfc54f">
然后记录一直在显示“已放弃本次探索”， 差不多每一秒刷一条
请问这种情况该怎么办，我用的是mumu模拟器，v5.9.0 beta

**Comment by Daydreamer114** [comment]
请在复现后前往主仓库提交 issue
